skops/io/_utils.py

```diff
@@ -58,9 +57,9 @@ def _import_obj(module, cls_or_func, package=None):
     return getattr(importlib.import_module(module, package=package), cls_or_func)
 
 
-def gettype(state):
-    if "__module__" in state and "__class__" in state:
-        return _import_obj(state["__module__"], state["__class__"])
+def gettype(module_name, cls_or_func):
+    if module_name and cls_or_func:
+        return _import_obj(module_name, cls_or_func)
```

Comment:
I wanted to check something here:

As it stands, is there any way loading a .skops file could redefine something in the global namespace?

My understanding (correct me if I'm wrong), is that this could only happen if:
* Code is defined that does that before calling `loads`
* Something does this in a user's imported module 

In either of those cases, this isn't a vulnerability with skops itself, so I think it's ok, but I wanted to make sure I've not missed somewhere that global namespaces could be changed during `load`, as that could lead to a vulnerability.

In other words, there's not currently a way someone could structure a `.skops` file that redefines a type we deem "trusted", like `np.random.Generator`, right?